docs/XBlocks.md

```diff
@@ -0,0 +1,217 @@
+# Group Project building blocks
+
+The following blocks are a part of Group Project XBlock v2.
+
+## Top-level XBlocks
+
+* ` Group Project` - top level XBlock, represents entire group project as a whole
+    * Settings:
+        * `Display Name` - human-friendly name for Group Project - displayed in admin console.
+    * Components: 
+        * `Group Project Activity` - multiple instances.
+        * [`Group Project Navigator`](#project-navigator-and-views) - single instance
+        * `Discussion` - external XBlock: see https://github.com/edx-solutions/xblock-discussion; single instance
+* `Group Project Activity` - represents single activity in group project, grading happens at this level, hence the block
+    carries settings for grading
+    * Components:
+        * `Display Name` - human-friendly activity name - appears in admin console and in Project Navigator (i.e. 
+            visible both to students and course staff)
+        * `Weight` - grading attribute - the higher the weight the more influential this activity compared to other
+            graded exercises in the same grading bucket (including group project activities and graded excercises)
```

Comment:
@e-kolpakov "this activity" ⟶ "this activity will be"; "excercises" ⟶ "exercises"